src/Dialogs/AbstractPropertiesDialog.vala

```diff
@@ -154,10 +154,25 @@ protected abstract class Files.View.AbstractPropertiesDialog : Granite.Dialog {
             uint64 fs_available = file_info.get_attribute_uint64 (FileAttribute.FILESYSTEM_FREE);
             uint64 fs_reserved = fs_capacity - fs_used - fs_available;
 
-            storagebar = new Files.StorageBar.with_total_usage (fs_capacity, fs_used + fs_reserved);
-            update_storage_block_size (fs_reserved, Files.StorageBar.ItemDescription.OTHER);
+            storage_levelbar = new Gtk.LevelBar.for_interval (0, fs_capacity) {
+                value = fs_used + fs_reserved,
+                hexpand = true
+            };
+            storage_levelbar.add_offset_value (Gtk.LEVEL_BAR_OFFSET_LOW, 0.6 * fs_capacity);
+            storage_levelbar.add_offset_value (Gtk.LEVEL_BAR_OFFSET_HIGH, 0.9 * fs_capacity);
+            storage_levelbar.add_offset_value (Gtk.LEVEL_BAR_OFFSET_FULL, fs_capacity);
+            storage_levelbar.get_style_context ().add_class ("inverted");
+
+            var storage_label = new Gtk.Label (
+                create_storage_label (fs_used + fs_reserved, fs_capacity)
+            );
 
-            info_grid.attach (storagebar, 0, line + 1, 4, 1);
+            info_grid.attach (storage_levelbar, 0, line + 1, 4);
+            info_grid.attach (storage_label, 0, line + 2, 4);
+
+            storage_levelbar.notify["value"].connect (() => {
+                storage_label.label = create_storage_label ((uint64) storage_levelbar.value + fs_reserved, fs_capacity);
+            });
```

Comment:
This is no longer needed (and the storage bar value already includes fs_reserved now)